t/md5.t

```diff
@@ -1,11 +1,20 @@
 use strict;
 use warnings;
 
-use Test::More tests => 3;
+use Test::More tests => 4;
+use File::Temp qw(tempfile);
 use Rex::Commands::MD5;
 
 my $test_file = Rex::Helper::File::Spec->catfile( 't', 'md5test.bin' );
 
+my ( $fh, $strangely_named_file ) = tempfile(
```

Comment:
Hmm, on second pass, I think it might be a good idea to add a short comment about why we name the file like this.

Reply:
I also included a comment about it.